test/integration-tests/test-server/test-server.ts

```diff
@@ -27,15 +26,30 @@ const customContext = (context: ExpressContext): Partial<TestContext> => {
     };
 };
 
-export async function startTestServer(): Promise<PolarisServer> {
+const getCustomContext = (
```

Comment:
why do you need this method?

Reply:
i need this method to distinguish between the tests with the custom context with warnings and the rest